src/pages/edge/adobe-journey-optimizer/proposition-edge-events.md

```diff
@@ -0,0 +1,354 @@
+                    ]
+                }
+            },
+            "data" : {
+                "__adobe" : {
+                    "ajo" : {
+                        "in-app-response-format" : 2
+                    }
+                }
+            },
+            "xdm" : {
+                "timestamp" : "2024-09-25T19:47:55.994Z",
+                "_id" : "4729BDF3-E155-4AB4-85F8-93F0DC15C92C",
+                "eventType" : "personalization.request"
+            }
+        }
+    ]
+}
+```
+
+### decisioning.propositionTrigger
+
+An event similar to the following is emitted any time a user has met the client-side criteria that qualifies them for a proposition:
+
+```json
+{
+    "events": [
+        {
+            "xdm": {
+                "_id": "F8B3C334-A67D-4174-A0F2-4E00523F5D02",
+                "eventType": "decisioning.propositionTrigger",
+                "timestamp": "2024-09-25T19:48:16.092Z",
+                "_experience": {
+                    "decisioning": {
+                        "propositionEventType": {
+                            "trigger": 1
+                        },
+                        "propositions": [
+                            {
+                                "id": "4b6fe796-3e4c-476d-be62-8610d851daef",
+                                "scope": "mobileapp://com.my.app.bundleIdentifier",
+                                "scopeDetails": {
+                                    "characteristics": {
+                                        "eventToken": "<BASE64_ENCODED_STRING>"
+                                    },
+                                    "activity": {
+                                        "id": "6aebeb06-faeb-4864-a27c-8be3efd6ff9b#876cb49a-7094-4f9e-9f9d-57ee26858462",
+                                        "matchedSurfaces": [
+                                            "mobileapp://com.my.app.bundleIdentifier/"
+                                        ]
+                                    },
+                                    "correlationID": "7d0408d2-e9ed-4d66-b0a5-2659fc37e2a9-0",
+                                    "decisionProvider": "AJO"
+                                }
+                            }
+                        ]
+                    }
+                }
+            }
+        }
+    ]
+}
+```
+
+### decisioning.propositionDisplay
+
+An event similar to the following is emitted any time a proposition is displayed to the application's UI:
+
+```json
+{
+    "events": [
+        {
+            "xdm": {
+                "_id": "F8B3C334-A67D-4174-A0F2-4E00523F5D02",
+                "eventType": "decisioning.propositionDisplay",
+                "timestamp": "2024-09-25T19:48:16.092Z",
+                "_experience": {
+                    "decisioning": {
+                        "propositionEventType": {
+                            "display": 1
+                        },
+                        "propositions": [
+                            {
```

Comment:
we also include `items` which has the PropositionItem id and token used for tracking code-based experiences with embedded decision scopes. Do you think it might be good to add?
This wiki has a sample payload
https://wiki.corp.adobe.com/display/~swsaraf/Code-based+experiences+tracking+in+mobile+SDK

Reply:
We can also do it in a following PR, it might be out of scope for this one since you've mentioned these events are for IAM only